lib/node/spvnode.js

```diff
@@ -196,64 +179,11 @@ class SPVNode extends Node {
    * Scan for any missed transactions.
    * Note that this will replay the blockchain sync.
    * @param {Number|Hash} start - Start block.
-   * @param {Bloom} filter
-   * @param {Function} iter - Iterator.
-   * @returns {Promise}
-   */
-
-  async scan(start, filter, iter) {
-    const unlock = await this.scanLock.lock();
-    const height = this.chain.height;
-
-    try {
-      await this.chain.replay(start);
```

Comment:
I think this means we can remove `chain.replay()` and `_replay()` now as well? grep returns 0 results outside of that function definition.

Reply:
There is still usage of`replay` as part of `invalidate`.

Reply:
ah right, didn't grep directly for `_replay()`